### DISCUSSION TITLE
set writeBufferSize not working

### DISCUSSION THREAD

**Discussion by whohasthis** [logfellow/logstash-logback-encoder#731] [Q&A]
Hello there, we use logstash-logback-encode version 6.6 with logback version 1.2.3, and in the logback.xml file we use something like below to set the buffer size to 131072
  <appender name="stash" class="net.logstash.logback.appender.LogstashTcpSocketAppender">
      ...
      <writeBufferSize>131072</writeBufferSize>
  </appender>

but we are still seeing this warning message in the console log, the same as before the writeBufferSize is set to 131072
14:39:24,644 |-WARN in net.logstash.logback.appender.LogstashTcpSocketAppender[logstash] - Dropped 76001 events (and counting...) due to ring buffer at max capacity [8192]

Could anyone point a finger on where might be wrong to cause the writeBufferSize setting not working ? Thanks a lot.

**Comment by brenuart** [comment]
Could you please post an example of your configuration file?
